### PR TITLE
Stringer based

### PR DESCRIPTION
**Pull request by sidharthsapru**

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -1,19 +1,46 @@
...
 ESLINT=node_modules/.bin/eslint
 SLACK_NOTIFY=docker-compose -f docker-compose.slack.yml run --rm
+SET_CREDENTIALS=$(DC) -f docker-compose-deploy.yaml run --rm config
+STRINGER_BUILD=$(DC) -f docker-compose-deploy.yaml run --rm stringer build --tags lambda
+
+export STRINGER_VERSION := 1.3.250
+export DOCKER_NODE_IMAGE := node
+export DOCKER_NODE_VERSION := 8.4.0-wheezy
+export CLUSTER := services
+export AWS_REGION := us-east-1
```
Comment (skilledDeveloper) [inline makefile]: Is this required? where do we use it?

File: makefile [hunk 2]
```diff
@@ -1,19 +1,46 @@
...
+
 
 .DEFAULT_GOAL := test
 .PHONY: test
 
+login:
+	@docker login -u "$(DOCKER_USER)" -p "$(DOCKER_PASS)"
+
+deploy:
+	$(NODE) npm install serverless
```
Comment (skilledDeveloper) [inline makefile]: - what is `$(NODE) ` and where does it come from?
- why do we need to install sls here? isn't it already installed with `make install`?

File: makefile [hunk 3]
```diff
@@ -1,19 +1,46 @@
...
+	$(SET_CREDENTIALS)
+	$(STRINGER_BUILD)
+	$(NODE) $(SLS)  deploy --stage $(AWS_ACCOUNT)
+	$(NODE) $(SLS) prune -n 7
+
 clean:
-	rm -rf node_modules
```
Comment (skilledDeveloper) [inline makefile]: why is this removed?

File: makefile [hunk 4]
```diff
@@ -24,11 +51,16 @@ test:
 run:
 	sls webpack serve
 
-deploy_functions:
-	$(SLS) deploy --stage $(DEPLOY_ENV)
-	$(SLS) prune -n 7
+deploy-local:
```
Comment (skilledDeveloper) [inline makefile]: how is this used?
can we update readme instructions?

File: stringer_spec/conf/main.yaml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (skilledDeveloper) [inline stringer_spec/conf/main.yaml]: Do we need this file? what's the use?

File: makefile (hunk 1 above)
Comment (sidharthsapru) [reply makefile]: Still working on these. For local deployment we need some environment settings

File: makefile (hunk 2 above)
Comment (sidharthsapru) [reply makefile]: copying from devops
http://scribe.vevo.com/stringer/lambda

File: stringer_spec/conf/main.yaml (hunk 5 above)
Comment (sidharthsapru) [reply stringer_spec/conf/main.yaml]: its for credstash. 
http://scribe.vevo.com/stringer/lambda

File: makefile (hunk 3 above)
Comment (sidharthsapru) [reply makefile]: oops. will fix this. It was annoying locally
